employees/templatetags/data_structure_element_selectors.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+from typing import Dict
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def get_key_value(dictionary: Dict[Any, Any], index: Any) -> Any:
+    try:
```

Comment:
Why not `return dictionary.get(index, "")` ? And I think its faster by some micro time ;)